src/pages/Profile/Body/GeneralInfo/DetailsColumn/DetailsColumn.tsx

```diff
@@ -12,26 +15,42 @@ export default function DetailsColumn({ className = "" }) {
   return (
     <div className="flex flex-col gap-6 w-full">
       <Balances />
-
-      <div
-        className={`${className} flex flex-col gap-8 w-full lg:w-96 p-8 border border-gray-l4 rounded text-navy-d4 dark:bg-blue-d6  dark:text-white`}
-      >
-        {p.registration_number && (
-          <Detail title="registration no.">{p.registration_number}</Detail>
-        )}
-        {p.street_address && (
-          <Detail title="address">{p.street_address}</Detail>
-        )}
-        <Detail title="active in">
-          {isEmpty(active_in_countries)
-            ? p.hq_country
-            : active_in_countries.join(", ")}
-        </Detail>
-        <Tags {...p} />
-        {p.social_media_urls && (
-          <Socials social_media_urls={p.social_media_urls} />
+      <div className={`${className} w-full lg:w-96`}>
+        <div
+          className={`flex flex-col gap-8 w-full p-8 border border-gray-l4 rounded`}
+        >
+          {p.registration_number && (
+            <Detail title="registration no.">{p.registration_number}</Detail>
+          )}
+          {p.street_address && (
+            <Detail title="address">{p.street_address}</Detail>
+          )}
+          <Detail title="active in">
+            {isEmpty(active_in_countries)
+              ? p.hq_country
+              : active_in_countries.join(", ")}
+          </Detail>
+          <Tags {...p} />
+          {p.social_media_urls && (
+            <Socials social_media_urls={p.social_media_urls} />
+          )}
+          <DonateButton className="w-full" />
+        </div>
+        {!(p.claimed ?? true) && (
```

Comment:
```suggestion
        {p.claimed !== false) && (
```
nit: don't have to implement this, it's just an alternative way to write the check

Reply:
fixed https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2822/commits/f94f257a237908843cd6915f312475b4d74a14e3